coral/src/domain/schema-request/schema-request-api.ts

```diff
@@ -23,28 +22,29 @@ const createSchemaRequest = (
   );
 };
 
-type GetSchemaRequestsArgs = {
-  pageNumber?: number;
-  requestStatus: SchemaRequestStatus;
-};
-
 type GetSchemaRequestsQueryParams = ResolveIntersectionTypes<
   Required<
     Pick<
       operations["getSchemaRequestsForApprover"]["parameters"]["query"],
       "pageNo" | "requestStatus"
     >
-  >
+  > &
+    Pick<
+      operations["getSchemaRequestsForApprover"]["parameters"]["query"],
+      "env" | "topic"
+    >
 >;
 
-const getSchemaRequestsForApprover = ({
-  requestStatus,
-  pageNumber = 1,
-}: GetSchemaRequestsArgs): Promise<SchemaRequestApiResponse> => {
+const getSchemaRequestsForApprover = (
+  args: GetSchemaRequestsQueryParams
+): Promise<SchemaRequestApiResponse> => {
   const queryObject: GetSchemaRequestsQueryParams = {
-    pageNo: pageNumber.toString(),
-    requestStatus: requestStatus,
+    pageNo: args.pageNo,
+    requestStatus: args.requestStatus,
+    ...(args.topic && { topic: args.topic }),
```

Comment:
Pretty wild that this works, but apparently spreading `false` is actually valid and behaves as we want here 🤯 

![Screenshot 2023-02-23 at 16 59 05](https://user-images.githubusercontent.com/20607294/220961695-f54decc7-c4a7-4c53-b59d-4d1fcabdd746.png)